schemas/test/en/test_supplementary_data_with_sds_schema_version.json

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
There's probably a good reason but just wanted to check - is it worth adding `sds_schema_version` to the existing `test_supplementary_data` schema instead?

Reply:
I believe it was because it meant 2 out of the 3 test sds datasets wouldn't open with the schema, as one was set to `v2` and the other `v3`, which would mean it wouldn't match with the `v1` set in the schema.

Reply:
Ah cool that makes sense, thanks 👍